src/protocols/strategies/convex.strategy.ts

```diff
@@ -85,7 +84,7 @@ export class ConvexStrategy {
   static async getValueSources(chain: Chain, vaultDefinition: VaultDefinition): Promise<CachedValueSource[]> {
     switch (vaultDefinition.vaultToken) {
       case TOKENS.BVECVX:
-        return retrieveBribesProcessorData(chain, vaultDefinition);
+        return retrieveHarvestForwarderData(chain, vaultDefinition);
       case TOKENS.BCRV_CVXBVECVX:
         return getLiquiditySources(chain, vaultDefinition);
```

Comment:
We need to figure out `retrieveHarvestForwarderData` here as well, there will be events distributing to it as well